README.md

```diff
@@ -238,3 +238,43 @@ Notes:
 * This API is not defined by O2ims Interface Specification.
 * The server supports the `alarmProbableCauses` endpoint for exposing a custom list of probable causes.
 * The list is available in [data folder](internal/files/alarms/probable_causes.json). Can be customized and maintained as required.
+
+#### Alarm Subscription server
+
+The alarm subscription server needs to connect to the non-kubernetes API of the
```

Comment:
I think isn't completely accurate: the alarm subscription server doesn't need to connect to the non-kubernetes API of the global hub. It only connects to the Kubernetes API to read and write the configmaps it uses to store subscriptions. I think you can remove this first sentence.